bench/frontend/playwright.config.ts

```diff
@@ -0,0 +1,32 @@
+import { defineConfig, devices } from '@playwright/test';
+import path, { dirname } from 'path';
+
+import { PluginOptions } from '@grafana/plugin-e2e';
+
+const testDirRoot = 'e2e/plugin-e2e/';
+
+export default defineConfig<PluginOptions>({
+  fullyParallel: true,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  reporter: 'html',
+  use: {
+    baseURL: process.env.GRAFANA_URL || `http://${process.env.HOST || 'localhost'}:${process.env.PORT || 3000}`,
+    trace: 'retain-on-failure',
+    httpCredentials: {
+      username: 'admin',
+      password: 'admin',
+    },
+    provisioningRootDir: path.join(process.cwd(), process.env.PROV_DIR ?? 'conf/provisioning'),
+  },
+  projects: [
+    // Login to Grafana with admin user and store the cookie on disk for use in other tests
+    {
+      name: 'authenticate',
+      testDir: `${dirname(require.resolve('@grafana/plugin-e2e'))}/auth`,
+      testMatch: [/.*\.js/],
+    },
```

Comment:
To execute the tests in dashboard.spec.ts, you'd need to add another Playwright project that runs these tests.